internal/adapters/entrypoints/rest/handlers/get_reports_pegin.go

```diff
@@ -0,0 +1,56 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/rsksmart/liquidity-provider-server/internal/adapters/entrypoints/rest"
+	"github.com/rsksmart/liquidity-provider-server/internal/usecases/pegin"
+	"github.com/rsksmart/liquidity-provider-server/pkg"
+	"net/http"
+)
+
+// NewGetReportsPeginHandler
+// @Title Get Pegin Reports
+// @Description Get the last pegins on the API. Included in the management API.
+// @Param GetReportsPeginRequest body pkg.GetReportsPeginPegoutRequest true "Date range for the report with startDate and endDate"
+// @Success 200 pkg.GetPeginReportResponse
+// @Route /reports/pegin [get]
+func NewGetReportsPeginHandler(useCase *pegin.GetPeginReportUseCase) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		var requestBody pkg.GetReportsPeginPegoutRequest
```

Comment:
again, a GET request shouldn't have a body https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/GET

Reply:
use the query parameters